docs/people-and-groups/google-sign-in.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Google Sign-In
+redirect_from:
+  - /docs/latest/administration-guide/10-single-sign-on
+---
+
+# Google Sign-In
+
+Enabling [Google Sign-In](https://developers.google.com/identity/sign-in/web/sign-in) or [LDAP](https://www.metabase.com/glossary/ldap) for single sign-on (SSO) lets your team log in with a click instead of using email and password. SSO can also be used to let people create Metabase accounts without asking an admin to add each person manually. You can find SSO options under **Settings** > **Admin settings** > **Authentication**.
```

Comment:
```suggestion
Enabling [Google Sign-In](https://developers.google.com/identity/sign-in/web/sign-in) for single sign-on (SSO) lets your team log in with a click instead of using email and password. SSO can also be used to let people create Metabase accounts without asking an admin to add each person manually. You can find SSO options under **Settings** > **Admin settings** > **Authentication**.
```